MaterialLib/MPL/Properties/CreateExponential.cpp

```diff
@@ -43,6 +43,9 @@ std::unique_ptr<Exponential> createExponential(
     auto const factor =
         //! \ogs_file_param{properties__property__Exponential__exponent__factor}
         exponent_data_config.getConfigParameter<double>("factor");
+    auto const offset =
```

Comment:
The offset should not be in the exponent specification.

Reply:
Thanks! Fixed. (I hope ;)